scripts/ci/Dockerfile.bitbake

```diff
@@ -0,0 +1,37 @@
+FROM debian:stable
+LABEL Description="Image for bitbaking"
+
+RUN sed -i 's#deb http://deb.debian.org/debian stable main#deb http://deb.debian.org/debian stable main contrib#g' /etc/apt/sources.list
+RUN sed -i 's#deb http://deb.debian.org/debian stable-updates main#deb http://deb.debian.org/debian stable-updates main contrib#g' /etc/apt/sources.list
+RUN apt-get update -q && apt-get install -qy \
+     build-essential \
+     bzip2 \
+     chrpath \
+     cpio \
+     default-jre \
+     diffstat \
+     gawk \
+     gcc-multilib \
+     git-core \
+     iputils-ping \
+     iproute \
+     libpython-dev \
+     libsdl1.2-dev \
+     locales \
+     procps \
+     python \
+     python3 \
+     python3-pexpect \
+     python3-requests \
```

Comment:
Do we really need python3-requests? I thought we gave up on using that.

Also, requiring python, python3, and libpython-dev is pretty annoying, but if that's what we need, then so be it.

Reply:
Actually, I adapted a lot of the stuff from https://github.com/advancedtelematic/dockerfiles/blob/master/bitbakery/Dockerfile. I think you're right about python3-requests, which was needed for a brief period.